dev/ToastNotifications/NotificationProperties.cpp

```diff
@@ -130,10 +130,10 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationProperties::get_ExpiresOnReboot(_Ou
 
 STDMETHODIMP_(HRESULT __stdcall) NotificationProperties::get_ToastProgressData(_Out_ ToastABI::IToastProgressData** progressData) noexcept
 {
-    // TODO: Uncomment once ToastProgressData is implemented
-    // auto lock = m_lock.lock_shared();
-    // *toastProgressData = m_toastProgressData.get();
-    *progressData = nullptr;
+    auto lock = m_lock.lock_shared();
```

Comment:
NIT: `auto lock{ m_lock.lock_shared() };`

[ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

same comment applies throughout